modules/reactor/test/tests/kernels/simple_diffusion/simple_diffusion.i

```diff
@@ -0,0 +1,44 @@
+[Mesh]
```

Comment:
Can you please remove the `modules/reactor/test/test/kernels` directory?

Reply:
sure